mail_tracking/models/mail_alias.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+
+
+class MailAlias(models.Model):
+    _inherit = 'mail.alias'
+
+    @api.model
+    @tools.ormcache('email')
+    def is_email_address_in(self, email):
+        name, domain = email.split('@')
+        aliases = self.search([
+            ('alias_name', '=', name),
+        ])
+        return any(aliases.mapped(lambda x: x.alias_domain == domain))
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        self.clear_caches()
+        return res
+
+    @api.multi
+    def write(self, vals):
+        res = super().write(vals)
+        if 'alias_name' in vals:
```

Comment:
```suggestion
        if 'alias_name' in vals or 'alias_domain' in vals:
```

Reply:
'alias_domain' It's an computed field without 'inverse' & 'search'. So... if you change these domain you don't need restart odoo instance?

Reply:
Oh OK, it was just by reading this code, I didn't know that.